src/main/java/com/lambdaschool/wellness/model/User.java

```diff
@@ -29,33 +30,69 @@
 
     @JsonIgnore
     private String password;
+
+    @ManyToMany
+    @JoinTable(name = "user_group_competition",
+            joinColumns = {@JoinColumn(name = "userid")},
```

Comment:
joinColumns needs to reference "id" column on user table. `@JoinColumn(name = "userid", referencedColumnName = "id")`